src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,28 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    userId,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      { 'TodoInfo--completed': completed === true },
+    )}
```

Comment:
`completed` it's a boolean type, so, no need to do as you did
```suggestion
    <article className={classNames('TodoInfo', {
       'TodoInfo--completed': completed
     })}
```